packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -96,6 +96,16 @@ class RequestWrapper {
       if (this.pluginCallbacks.cacheWillUpdate.length !== 1) {
         throw ErrorFactory.createError('multiple-cache-will-update-plugins');
       }
+      this.cacheableResponseCheck = this.pluginCallbacks.cacheWillUpdate[0];
```

Comment:
Sorry but I don't follow the differences between this.cacheableResponseCheck and this._defaultCacheableResponseCheck where default isn't static and IF it's set it's another weird binding thing.